### ISSUE TITLE
Stuck at “right before trigger”

### ISSUE THREAD

**Issue by FnGAnomymous** [checkra1n/BugTracker#885]
**Tell us about your setup:**
1. What iDevice are you using? - iPhone X 
2. On what version of iOS is it? - 13.3
3. What version of checkra1n are you using? GUI or CLI? - 0.9.8 CLI
4. What is your host system (OS version? Hackintosh? VM? etc.)? - Windows 7 booted into           Ubuntu 
5. How are you connecting to the device (USB-A? USB-C? Apple/3rd party cable? Through a USB hub?)? - Apple cable

**What are the steps to reproduce the issue?**
1. boot linux mint/others
2. run checkra1n
3. follow steps on the screen
4. get stuck
...

**What do you expect, and what is happening instead?**
 Expecting to Jailbreaks but the progress bar gets stuck

**Does the issue also occur if you enable Safe Mode?**
   Haven’t tried that

**Any other info, error logs, screenshots, ...?**
![image](https://user-images.githubusercontent.com/47499423/74271561-1a694980-4d33-11ea-85ba-c9e4734a6882.jpg)

**Comment by ann0see** [comment]
Did you install usbmuxd via apt install usbmuxd (as root)

**Comment by jaylex32** [comment]
> Did you install usbmuxd via apt install usbmuxd (as root)

Would this fix this error? and why?

**Comment by ann0see** [comment]
Maybe it would fix the problem since usbmuxd basically is the driver for iDevices on Linux.

**Comment by FDeghy** [comment]
i installed this but not fix this error(stuck)

**Comment by ann0see** [comment]
Did you try an other OS? 
If it doesn’t work, probably it’s the same or a similar problem like the raspberry pi 3 and lower have.

**Comment by KimChow** [comment]
I can using the ubuntu to Jailbreaks ipad mini 4.

But I got the same wrong with ipad mini 2.

**Comment by KimChow** [comment]
I have using ubuntu 18.

**Comment by ann0see** [comment]
So the iPad mini 4 works but the iPad mini 2 doesn’t?

**Comment by KimChow** [comment]
Yes. I don't know why. How can I post more wrong infomation for you?

ipad mini 4 is ios 13.3.1
ipad mini 2 is ios 12.4.4.

I want jailbreak my ipad mini 2.

**Comment by FnGAnomymous** [comment]
Update! Jail broke my device successfully few days ago but last night Cydia unexpectedly crashed and I can't reopen Cydia anymore I tried restoring my device from the checkra1n app installed on my iPhone #775

**Comment by AMDBartek** [comment]
This is a known issue. #984

**Comment by rly907** [comment]
#1169 like this

**Comment by jacobecret** [comment]
Okay well that picture isnt cli mode.. thats gui mode.
Run it as cli "checkra1n -c" and put your phone into dfu mode before hand and see if you get the same results.

**Comment by jacobecret** [comment]
> So the iPad mini 4 works but the iPad mini 2 doesn’t?

i can personally tell you the ipad mini 2 works. [
![IMG_2486](https://user-images.githubusercontent.com/65593939/82378438-f5434180-99d9-11ea-80f6-ac9c499b37fc.jpg)
![IMG_2487](https://user-images.githubusercontent.com/65593939/82378442-f70d0500-99d9-11ea-8fed-9571e4feeba9.jpg)

**Comment by rly907** [comment]
ipad mini 2 works but its just very slow (and some apps like crashing a lot)

or at least it was for me
